.travis.yml

```diff
@@ -21,11 +21,17 @@ jobs:
         - yarn build
         - yarn run styleguide:build
       deploy:
-        provider: pages
-        skip_cleanup: true
-        commiter_from_gh: true
-        github_token: $GITHUB_TOKEN
-        on:
-          branch: master
-        target_branch: gh-pages
-        local_dir: docs
+        - provider: pages
```

Comment:
Try indenting the `provider` items left (including the hyphen) so that they align with the same column as the `deploy` item. That should do it.